Orochi/OrochiUtils.cpp

```diff
@@ -381,24 +381,45 @@ struct OrochiUtilsImpl
 	}
 };
 
-char* OrochiUtils::s_cacheDirectory = "./cache/";
+OrochiUtils::OrochiUtils() 
+{
+	m_cacheDirectory = "./cache/";
+}
```

Comment:
```suggestion

```
Reason: we can default initialize directly on the member variables and do not need the explicitly defined constructor.